MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include versioneer.py
+include lib/nc_time_axis/_version.py
```

Comment:
Incidentally, the lib is pretty superfluous, and is generally not used in any of our recent repos.